pmm/aws-staging-start.groovy

```diff
@@ -231,7 +231,7 @@ pipeline {
                             echo '$SSH_KEY' >> /home/ec2-user/.ssh/authorized_keys
                         fi
 
-                        sudo yum -y install https://repo.percona.com/yum/percona-release-1.0-25.noarch.rpm
+                        sudo yum -y install https://repo.percona.com/yum/percona-release-1.0-27.noarch.rpm
```

Comment:
it is better to just use latest version of percona-release https://repo.percona.com/yum/percona-release-latest.noarch.rpm